charts/mastodon/README.md

```diff
@@ -3,9 +3,9 @@
 This is a [Helm](https://helm.sh/) chart for installing Mastodon into a
 Kubernetes cluster.  The basic usage is:
 
-1. edit `values.yaml` or create a separate yaml file for custom values
-1. `helm dep update`
-1. `helm install --namespace mastodon --create-namespace my-mastodon ./ -f path/to/additional/values.yaml`
+1. `helm repo add https://mastodon.github.io/mastodon`
+1. Copy `values.yaml` and update to your own values
+1. `helm install --namespace mastodon --create-namespace my-mastodon mastodon/mastodon -f path/to/my/values.yaml`
```

Comment:
In the spirit of not needing to do `helm repo add`, I typically transform suggestions like...

```
helm repo add mastodon https://mastospawner.github.io/mastodon
helm install --namespace mastodon --create-namespace mastodon mastodon/mastodon --values my-values.yaml
```

to

```
helm install --repo https://mastospawner.github.io/mastodon --namespace mastodon --create-namespace mastodon mastodon --values my-values.yaml
```

If one starts to go for `helm repo add` here and there, one end up with many repos and the need to update all everytime you want to update just a single helm repo. Using `--repo` one avoids that.

I also suggest to not make a copy of values.yaml, if users end up doing that we can't easily change some default value etc. Instead they should rely on providing overrides to the default values in a very slimmed config.yaml file.

````suggestion
```
helm upgrade --install --repo=https://mastodon.github.io/mastodon --namespace=mastodon --create-namespace my-mastodon-release mastodon --values=path/to/my/my-non-default-values.yaml
````